tests/e2e/nebulacluster/nebulacluster.go

```diff
@@ -49,50 +50,66 @@ var _ = ginkgo.Describe("NebulaCluster", func() {
 	})
 
 	ginkgo.Describe("difference spec", func() {
-		kruiseReference := v1alpha1.WorkloadReference{
-			Name:    "statefulsets.apps.kruise.io",
-			Version: "v1alpha1",
-		}
-		nebulaSchedulerName := "nebula-scheduler"
+		var (
+			kruiseReference = v1alpha1.WorkloadReference{
+				Name:    "statefulsets.apps.kruise.io",
+				Version: "v1alpha1",
+			}
+			nebulaSchedulerName = "nebula-scheduler"
```

Comment:
How about providing a option for scheduler?

Reply:
Both need to be tested. The default is `default-scheduler`, and some test cases use `nebula-scheduler`.